src/pages/Auth/Login/index.tsx

```diff
@@ -54,31 +71,77 @@ export default function Login(): JSX.Element {
       isAutoLoginChecked: false,
     },
   });
-  const submitLogin = useLoginRequest();
+
+  const [error, setError] = useState<boolean>(false);
+  const submitLogin = useLoginRequest({ onError: setError });
```

Comment:
`useBoolean` util hook을 만들어두었습니다.
```suggestion
  const { value: error, setTrue: catchError } = useBoolean(false);
  const submitLogin = useLoginRequest({ onError: catchError });
```

Reply:
수정했습니다.[48c82af](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/43/commits/48c82afba5c6f5ea99b127595f871316215cbfec)